spec/chef/helper_spec.rb

```diff
@@ -5,18 +5,13 @@
   include ChefVault::Mixin::Helper
 
   let(:json_data) { '{"username": "root", "password": "abcabc"}' }
-  let(:json_data_control_char) { '{"username": "root", "password": "abc\abc"}' }
   let(:buggy_json_data) { '{"username": "root", "password": "abc\abc"' }
```

Comment:
will be good to add tests that ensure we allow for tabspace, whitespace and newline going forward.